### PR TITLE
Timeline relations bug relations folding

### PR DESCRIPTION
**Pull request by romanroe**

### CODE REVIEW

**Comment by romanroe** [comment]
@oliverguenther I forgot to remove some console.error statements. Changes are pushed, CI skipped.

File: frontend/app/components/wp-table/timeline/global-elements/wp-timeline-relations.directive.ts [hunk 1]
```diff
@@ -168,32 +168,37 @@ export class WorkPackageTableTimelineRelations {
...
 
   private update() {
-    this.removeAllVisibleElements();
-    this.renderElements();
+    setTimeout(() => {
+      this.removeAllVisibleElements();
+    }, 2000);
+    setTimeout(() => {
+      this.renderElements();
+    }, 4000);
```
Comment (oliverguenther) [inline frontend/app/components/wp-table/timeline/global-elements/wp-timeline-relations.directive.ts]: You'll want to remove this, right? ;-)

Comment (romanroe) [reply frontend/app/components/wp-table/timeline/global-elements/wp-timeline-relations.directive.ts]: code + 35° Celsius === crappy code

**Comment by romanroe** [comment]
@oliverguenther Good to merge

**Pull request merged**